### DISCUSSION TITLE
Issues with extension updating

### DISCUSSION THREAD

**Discussion by Chikorita-Lover** [ScratchAddons/ScratchAddons#2002] [Help] [answered]
Hello, recently I saw that version 1.12.0 was released for Scratch Addons and I was very excited! However, my extension doesn't seem to want to update. How would I manually update the extension? It's currently on version 1.11.2, and my Chrome installation is on version 89.0.4389.82.

**Comment by mxmou** [comment, marked as answer]
I don't think manual updating is possible in Chrome.

**Reply by Chikorita-Lover** [reply]
Alright, but do you know how I could solve the issue?

**Reply by mxmou** [reply]
Chrome frequently updates extensions automatically.

**Reply by Chikorita-Lover** [reply]
Alright, thanks!

**Reply by retronbv** [reply]
> I don't think manual updating is possible in Chrome.

Actually, not true. You could side-load the extension, but then it will not auto update.

**Comment by WorldLanguages** [comment]
Go to `chrome://extensions`, enable developer mode, click the new "update" button, then disable developer mode

**Reply by Chikorita-Lover** [reply]
Seems the extension's already updated itself, but I'll definitely try that in the future!
